src/API/Google/AdsCampaign.php

```diff
@@ -415,25 +418,26 @@ protected function combine_campaigns_and_campaign_criterion_results( array $camp
 			// negative: Whether to target (false) or exclude (true) the criterion.
 			->where( 'campaign_criterion.negative', 'false', '=' )
 			->where( 'campaign_criterion.status', 'REMOVED', '!=' )
+			->where( 'campaign_criterion.location.geo_target_constant', '', 'IS NOT NULL' )
 			->get_results();
 
 		/** @var GoogleAdsRow $row */
 		foreach ( $campaign_criterion_results->iterateAllElements() as $row ) {
-			$campaign           = $row->getCampaign();
-			$campaign_id        = $campaign->getId();
-			$campaign_criterion = $row->getCampaignCriterion();
-			$location           = $campaign_criterion->getLocation();
-
-			// TODO: Convert the geo_traget_constant to the ISO 3166-1 alpha-2 country code
-			// after https://github.com/woocommerce/google-listings-and-ads/issues/1229 is done.
-			$geo_target_constant = $location ? $location->getGeoTargetConstant() : null;
+			$campaign    = $row->getCampaign();
+			$campaign_id = $campaign->getId();
 
 			if ( ! isset( $campaigns[ $campaign_id ] ) ) {
 				continue;
 			}
 
-			if ( ! empty( $geo_target_constant ) ) {
-				$campaigns[ $campaign_id ]['targeted_locations'][] = $geo_target_constant;
+			$campaign_criterion  = $row->getCampaignCriterion();
+			$location            = $campaign_criterion->getLocation();
+			$geo_target_constant = $location->getGeoTargetConstant();
+			$location_id         = $this->parse_geo_target_location_id( $geo_target_constant );
+			$country_code        = $this->google_helper->find_country_code_by_id( $location_id );
+
+			if ( isset( $country_code ) ) {
```

Comment:
Just personal preference: `isset` is technically the correct way to identify it isn't `null`. Although the function definition doesn't guarantee that it can't return an empty string `''`. So I would have checked for both:

```suggestion
			if ( $country_code ) {
```

Reply:
Good point, thanks! Updated in 46e5f7c.